### DISCUSSION TITLE
Keep buffer listed when leaving buffers of 0Glog

### DISCUSSION THREAD

**Discussion by dkdog** [tpope/vim-fugitive#1880] [Q&A] [answered]
A buffer opened via `0Glog` automatically gets deleted (becomes unlisted) if I merely switch to a different buffer.

Can I make vim-fugitive NOT do that?

Let's say I view my current file at a previous commit, I open `:0Glog` and I go to the file at a previous commit.

I want to keep that content around. But the moment I switch to a different buffer. That buffer becomes unlisted.

There is some subtle usability here.

For example, my cursor position is lost if I reopen a unlisted buffer. I have `:set nostartofline` usually.

**Comment by tpope** [comment, marked as answer]
This will respect `'hidden'`:

```vim
autocmd User FugitiveObject setlocal bufhidden=
```

You can also force to hide regardless with `bufhidden=hide`.

*Edited to use `FugitiveObject` rather than the more narrowly targeted `FugitiveCommit`.*

**Comment by Mo3m3n** [comment]
I came through a similar issue where I need to keep buffers opened by Gedit, Gsplit, etc 
It seems the event name changed and using the event `Fugitive` instead of `FugitiveCommit` worked for me (after looking for the correct name in "plugin/fugitive.vim" )
Maybe it is worth mentioning and maintaining such config in the doc.
Thanks for this nice plugin.

**Reply by tpope** [reply]
The event name didn't change. `Fugitive` is the wrong event. Also it's deprecated.

All events are documented under `:help fugitive-autocommands`. You might be interested in `FugitiveObject`.

**Reply by Mo3m3n** [reply]
Right the `FugitiveBlob` is what am looking for, `FugitiveCommit` is just a different even that is not triggered in my case.
I have missed that part of the doc. Thanks !
